tools/c7n_gcp/c7n_gcp/resources/resourcemanager.py

```diff
@@ -410,6 +410,39 @@ def get_item_values(self, resource):
         for page in pages:
             contacts.extend(page.get('contacts', []))
         return contacts
+    
+@Organization.filter_registry.register('org-policy')
+class OrgPoliciesFilter(ListItemFilter):
```

Comment:
This is a clever idea - extending `ListItemFilter` rather than `ValueFilter` for custom filters that expect to return a list :bulb: